src/components/useGameState.tsx

```diff
@@ -40,6 +40,22 @@ export const GameStateProvider: React.FC<{ children?: ReactNode }> = ({
     setGameState(initialGameState);
   };
 
+  // Expose game state functions to window for testing and debugging
+  React.useEffect(() => {
```

Comment:
import `useEffect` directly, don't reference it from `React`. `React` is only in the global namespace for the sake of TSX.